pkg/k8s/diverts/translate.go

```diff
@@ -72,22 +70,48 @@ func translateService(m *model.Manifest, s *apiv1.Service, resourceVersion strin
 	return result
 }
 
-func translateEndpoints(m *model.Manifest, e *apiv1.Endpoints, resourceVersion string) *apiv1.Endpoints {
+func translateEndpoints(m *model.Manifest, s *apiv1.Service) *apiv1.Endpoints {
 	result := &apiv1.Endpoints{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:            e.Name,
-			Labels:          e.Labels,
-			Annotations:     e.Annotations,
-			ResourceVersion: resourceVersion,
+			Name: s.Name,
+			Labels: map[string]string{
+				model.DeployedByLabel: m.Name,
+			},
+			Annotations: map[string]string{
+				model.OktetoAutoCreateAnnotation: "true",
+			},
+		},
+		Subsets: []apiv1.EndpointSubset{
```

Comment:
This is the main change. We use the target service ip, instead of the target pod ips.